oi/type_graph/TypeIdentifier.cpp

```diff
@@ -73,10 +73,17 @@ void TypeIdentifier::visit(Container& c) {
       for (const auto& info : passThroughTypes_) {
         if (std::regex_search(paramClass->fqName(), info.matcher)) {
           // Create dummy containers
-          auto& dummy =
-              typeGraph_.makeType<Container>(info, param.type().size());
-          dummy.templateParams = paramClass->templateParams;
-          c.templateParams[i] = dummy;
+          Container* dummy;
+          if (auto it = passThroughTypeDummys_.find(paramClass->id());
```

Comment:
Could you add a comment and a unit test explaining the purpose of this new code please?